webpack/react_app/components/TargetingHosts/TargetingHostsActions.js

```diff
@@ -1,8 +1,6 @@
-import { getURI } from 'foremanReact/common/urlHelpers';
 import { get } from 'foremanReact/redux/API';
 import { withInterval } from 'foremanReact/redux/middlewares/IntervalMiddleware';
 import { TARGETING_HOSTS } from './TargetingHostsConsts';
 
-const url = getURI().addQuery('format', 'json');
-export const getData = () =>
+export const getData = url =>
```

Comment:
I think we can move this method to the `index.js` file? since it's already using the middleware helpers.
after that you can also delete this file :)

Reply:
Can I keep it there? I have another PR where I extend this file with more functions.

Reply:
forget what I said then :)